hw11.py

```diff
@@ -0,0 +1,53 @@
+import time
+
+
+def name_and_time(func):
+    print(f'You called function "{func.__name__}" on {time.ctime()}')
+
+    def inner_deco_func():
+        func()
```

Comment:
It is better to use `return` statement here since decorated functions can return some value